src/main.js

```diff
@@ -1,30 +1,26 @@
-const http = require("http");
+'use strict';
+const express = require('express');
+const bodyParser = require('body-parser')
 const comment = require("./comment.js");
 
-const host = 'localhost';
-const port = 8000;
+const PORT = 8080;
```

Comment:
Either this needs changing back to `8000` or the command in the README needs updating to `8080` 😁

Reply:
other than that it's all good I think 😄

Reply:
Done!

There is another PR incoming with the dockerfile though :)